esp/applications/eclwatch/eclwatch.yaml

```diff
@@ -34,6 +34,11 @@ eclwatch:
         xslt:
         - name: def_file
           ^: "./smc_xslt/def_file.xslt"
+   WsDfs:
+      DisableUppercaseTranslation: false
+      ViewTimeout: 1000
+      EclServer: ''
+      EclServerQueue: ''
```

Comment:
NB: not entirely sure if this section has to be here at all

Reply:
These were mapped from the existing bare metal config in an automated way without actually diving into the individual services code.  Bare metal config might still have remnants of old features that were never cleaned up (or similar).  At some point they should be cleaned up directly, but we can certainly remove ones that aren't used as we come across them.

Reply:
My comment was based on existing services.  For new services only add things here that you want to be service specific values that you want the user to be able to override.  The defaults go here.  Technically you could leave off the defaults if they were set in code, but people do use this file as a quick reference of fields and their default values as well.

Reply:
again, there are no current config options, but perhaps best to have an entry as a placeholder/indicator for future use, so I'll remove the irrelevant values but leave WsDfs: only.